src/server/c-interop-native-extensions.md

```diff
@@ -10,7 +10,7 @@ May 2012_
 The extension mechanism discussed in this page is for deep integration of the VM
 by custom Dart embedders.
 This extension mechanism is **unsupported** for extension purposes,
-and may be removed in a future release.
+and is **[being removed](https://github.com/dart-lang/sdk/issues/45451)** over the next few future releases.
```

Comment:
```suggestion
and is **[being removed](https://github.com/dart-lang/sdk/issues/45451)**.
```

Reply:
I think the bug you link to is clear enough.

Reply:
That makes sense, particularly since most users won't need to know more than that it is being removed.